### PR TITLE
a couple more things related to the tests

### PR DESCRIPTION
**Pull request by dashea**

### CODE REVIEW

File: tests/lib/shutup.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Author: David Shea <dshea@redhat.com>
+
+import os, contextlib
```
Comment (jkonecny12) [inline tests/lib/shutup.py]: I think it would be better to import only `contextmanager` from `contextlib`.

**Comment by jkonecny12** [comment]
Other than that small point I think it's ok.

**Comment by vpodzime** [comment]
Looks good to me too with the same suggestion as @jkonecny12

**Comment by dashea** [comment]
Modified and pushed.

**Pull request closed without merging**